docs/core/integration-quality-scale/rules/iqs051.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "IQS051 - The documentation provides troubleshooting information"
+---
+
+IQS051 is as follows:
+
+The documentation provides troubleshooting information
+
+## Reasoning
+
+We should give the user enough information to figure out themselves on how to fix a common issue.
+This decreases the amount of support requests and increases the user experience.
+
+## Example implementation
+
+```markdown
+## Troubleshooting
+
```

Comment:
```suggestion
## Can’t access Home Assistant in my browser

### Symptom: “This site can’t be reached”

When trying to access Home Assistant in the browser, the browser shows the message “This site can’t be reached”.

#### Description

This means the browser can’t find your Home Assistant installation on the network.

#### Resolution

To resolve this issue, try the following steps:

1. Make sure your Home Assistant device is powered up (LEDs are on).
2. Make sure your Home Assistant installation is connected to the internet:
   - Make sure the Ethernet cable is plugged-in to both Home Assistant and to your router or switch.
   - Make sure your network has internet access.
3. ....
 
 #### Symptom: “ERR_CONNECTION_REFUSED”
 
 When trying to access Home Assistant in the browser, the browser shows the message “This site can’t be reached”.

#### Description

This could mean that there was a typo in the port part of the URL

#### Resolution: Check the URL
 
1. Check the spelling of the URL.
    - Typically, the URL is [http://homeassistant.local:8123](http://homeassistant.local:8123).
2. If you are running an older Windows version or have a stricter network configuration, try [http://homeassistant:8123](http://homeassistant:8123) instead.
```

I've added a bit of a longer example that has different symptoms for "the same" problem. I'm sure there are troubleshooters that can be dealt with a 2-liners. However, this example could provide the structure we can use if there is a bit more flesh on the bone.

Reply:
Although I do like the idea, I did change it to talk more about a device because that is what we expect from an integration quality. I did not have any inspiration for a second symptom, so I might revisit this later, but if you have one, feel free to comment :)